packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -117,7 +117,7 @@
   .container {
     padding: var(--calcite-spacing-sm);
   }
-  .column,
+  .column.has-content,
```

Comment:
To reduce specificity and for consistency, can we change the state modifiers to be more BEM-like? For example:

```scss
.column--has-content,
.row { /* ... */ }
```

Applies to other state modifier classes.